tests/rewards/test_reward_fn.py

```diff
@@ -7,7 +7,7 @@
 
 OBS = np.random.randint(0, 10, (64, 100))
 ACTS = NEXT_OBS = OBS
-DONES = np.zeros(64, dtype=np.bool)
+DONES = np.zeros(64, dtype=np.bool)  # type: ignore
```

Comment:
This fixes it and I think is the intended NumPy use:
```suggestion
DONES = np.zeros(64, dtype=np.bool_)
```

Reply:
Yes, that is correct. I learned this later but forgot to go back. Thanks for pointing it out. Another option is to do `np.dtype('bool')`.